index.html

```diff
@@ -0,0 +1,214 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Portifolio mobile</title>
+  <link rel="stylesheet" href="index.css" />
+</head>
+
+<body>
+  <div class="font">
+    <section class="first-section">
```

Comment:
- [ ] I recommend using the `section` tag for the main container that contains all of the cards and using the `div`  tag for each individual card.